index.html

```diff
@@ -41,43 +40,43 @@ <h5 class="connect">LET'S CONNECT</h5>
         </div>
         </section>
       </main>
-
-
-<!--<section class="container1">
-  <div class="card">
-  </div>
-</section> -->
-
+<!-- End Main Page  -->
+<!--  Start  Page 1 -->
       <section class="container1">
-          <div class="card">
+          <div class="card1">
              <div class="portfolio"><img src="./img/Snapshoot-Portfolio.png" alt="portfolio"></div>
              <div class="portfolio1"><img src="./img/Snapshoot-Portfolio-big.png" alt="portfolio1"></div>
 
              <div class="contenido1">
-          <h2 class="tonic">Tonic</h2>
-          <ul class="info">
-              <li class="tonic-detail">CANOPY</li>
-              <li><img src="./img/Counter.png" alt="punto"></li>
-              <li class="tonic-detail2">Back End Dev</li>
-              <li><img src="./img/Counter.png" alt="counter"></li>
-              <li class="tonic-detail2">2015</li>
-          </ul>
-          <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <div>
+               <h2 class="tonic">Tonic</h2>
+                <ul class="info">
+                  <li class="tonic-detail">CANOPY</li>
+                  <li><img src="./img/Counter.png" alt="punto"></li>
+                  <li class="tonic-detail2">Back End Dev</li>
+                  <li><img src="./img/Counter.png" alt="counter"></li>
+                  <li class="tonic-detail2">2015</li>
+               </ul>
+              <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+            <div>
               <ul class="buttons">
                   <li><button type="button" class="int-button">html</button></li>
                   <li><button type="button" class="int-button">css</button></li>
                   <li><button type="button" class="int-button">JavaScript</button></li>
               </ul>
           </div>
-            <div><button type="button" class="see">See Project</button>
-           </div>
+            <div><button type="button" class="see">See Project</button> </div>
```

Comment:
- [ ] Please create the style for the button pressed following the Figma template. It's a project requirement.